website/homepage/src/docs/template.html

```diff
@@ -157,6 +164,25 @@ <h3 class="text-3xl font-light my-4" id="update-section-header">Update</h3>
               {% endwith %}
             </div>
           </section>
+      <section id="events" class="pt-2 flex flex-col gap-10 mb-8">
+        <div>
+          <h2 id="events-header"
+              class="text-4xl font-light mb-2 pt-2 text-orange-500">Events</h2>
+          <p class="mt-8 text-lg">
+           Events give you control of when your functions are run in the app. For example,
+           on a Button, the `click` event will trigger the function when the button is clicked. Each component
+           and many layouts have specific events that they support. You can find the events that a component supports
+           by going to its documentation.
+          </p>
+        </div>
+        {% for event in docs["events"] %}
+            {% include "docs/event_template.html" %}
+        {% endfor %}
+        <!-- Document the load event with the events as well-->
```

Comment:
I think the `load` event should go with the other events as well. I'm worried users will not know of the load event if it's only listed in the `Blocks` docs.